android/pytorch_android/src/main/cpp/pytorch_jni_common.cpp

```diff
@@ -10,6 +10,54 @@
 
 namespace pytorch_jni {
 
+bool Trace::is_initialized_ = false;
+
+#if defined(TRACE_ENABLED) && defined(__ANDROID__)
+Trace::fp_ATrace_beginSection Trace::ATrace_beginSection = {};
```

Comment:
I would drop the `= {}` here.  Static scalars are always initialized to null/0, and using `{}` with a scalar feels weird to me.  Just preference, though.